base/files/file_util.cc

```diff
@@ -219,6 +219,24 @@ bool ContentsEqual(const FilePath& filename1, const FilePath& filename2) {
   // We open the file in binary format even if they are text files because
   // we are just comparing that bytes are exactly same in both files and not
   // doing anything smart with text formatting.
+#ifdef STARBOARD
+  // std::ifstream doesn't work on all our platforms.
```

Comment:
Do we know that this is still true with our newer c++ standard requirements? If it's something we should come back to check, change the macro check to USE_HACKY_COBALT_CHANGES (probably with a comment to clarify what needs to be checked)